content/en/apps/tutorials/local-setup.md

```diff
@@ -37,18 +37,41 @@ To read more about these concepts, see our [Docker Setup guide]({{< relref "core
 
 Before you begin, you need to have some useful software and tools that are required for things to work.
 
-Install [nodejs](https://nodejs.org/en/) 8 or later and [npm](https://www.npmjs.com/get-npm). As well, [install `docker` and `docker-compose`]({{< relref "apps/guides/hosting/requirements#docker" >}}).
+Install [nodejs](https://nodejs.org/en/) 8 or later, [npm](https://www.npmjs.com/get-npm), [git](https://git-scm.com/downloads) or the [Github Desktop](https://desktop.github.com/) and [`docker` and `docker-compose`]({{< relref "apps/guides/hosting/requirements#docker" >}}).
 
 ## Implementation Steps
 
 Now that you have the dependent tools and software installed, you are ready to set up your CHT local environment.
  
 
-### 1. Install medic-conf
+
+### 1. Install the Core Framework
+
+Check out the [cht-core respository](https://github.com/medic/cht-core) to your local machine, either by using the [Github Desktop app](https://desktop.github.com/) or by running the following command in the directory you want to check the code out into: `git clone https://github.com/medic/cht-core.git`. This will create a `cht-core` directory.  
+
+Open your terminal or command line and navigate to `cht-core` directory which is where the `docker-compose.yml` file is. Run the command:
```

Comment:
Suggested changes, but also consider using terminal or command line (not both) unless you mean two different things:

```suggestion
Open your terminal or command line and navigate to the `cht-core` directory, where you should see the `docker-compose.yml` file. Run the command:
```

Reply:
agreed! standardized on "terminal"